openshift/registry.Dockerfile

```diff
@@ -10,7 +10,7 @@ FROM registry.ci.openshift.org/ocp/4.18:base-rhel9
 USER 1001
 COPY --from=builder /build/registry /registry
 COPY --from=builder /build/push /push
-COPY openshift/manifests /openshift/manifests
+COPY openshift/operator-controller/manifests /openshift/operator-controller/manifests
```

Comment:
Again, the manifest destination should remain at `/openshift/manifests`. Only the source should change.

Reply:
I see good point !!!